src/components/templates/ComponentWithNavbar.tsx

```diff
@@ -0,0 +1,15 @@
+import { Container } from "@mui/material";
+import { Navbar } from "../molecules/Navbar";
+
+interface IComponentWithNavbar {
+  children: React.ReactNode;
+}
+
+export const ComponentWithNavbar = ({ children }: IComponentWithNavbar) => {
```

Comment:
I don't think this component is necessary. You can put Navbar in root component so it shows up on every page and renders only once.